### ISSUE TITLE
Unhelpful error from event hookup

### ISSUE THREAD

**Issue by rchande** [dotnet/roslyn#7565]
Try to use event hookup on tab at the caret. If you typo-ed your event declaration like I did, you get an error about "couldn't create solution". Perhaps we could
1. Fail, but indicate why it failed
2. Generate an eventhandler with a bad signature (you get the same error if the type doesn't exist, so this might be useful for GFU scenarios).

``` C#
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

class Eventer
{
    public event EventArgs e;
}

class Program
{
    Eventer e = new Eventer();
    void Main(string[] args)
    {
        this.e.e +=$$
    }

}
```

![image](https://cloud.githubusercontent.com/assets/3751401/11878121/aed9fca0-a4a8-11e5-97f4-0ad2d1dbb3c2.png)

**Comment by dpoeschl** [comment]
The failure is happening here: http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp.EditorFeatures/EventHookup/EventHookupCommandHandler_TabKeyCommand.cs,256

We could definitely give a better error message here, but using System.EventHandler when the delegate type can't be inferred would be fine too.

**Comment by CyrusNajmabadi** [comment]
Closing out as not-repro.  We don't give any error now.  The experience seems fine to me.  THe code is in error and doesn't generate anything.
